app/helpers/application_helper.rb

```diff
@@ -212,4 +212,21 @@ def list_all_timezones
     TIMEZONES
   end
 
-end
+  #conference attendance helper
```

Comment:
If this should actually be part of a ConferenceAttendanceHelper I would probably move it to it's own helper class/file.